src/server/game/Handlers/GroupHandler.cpp

```diff
@@ -1162,121 +737,110 @@ void WorldSession::HandleRequestPartyMemberStatsOpcode(WorldPacket& recvData)
-    uint64 auramask = 0;
-    size_t maskPos = data.wpos();
-    data << uint64(auramask);                          // placeholder
-    data << uint32(MAX_AURAS);                         // count
+    partyMemberState.PowerType = uint8(powerType);
+    partyMemberState.Status = playerStatus;                                 // GROUP_UPDATE_FLAG_STATUS
+    partyMemberState.CurrentHealth = uint32(player->GetHealth());           // GROUP_UPDATE_FLAG_CUR_HP
+    partyMemberState.MaxHealth = uint32(player->GetMaxHealth());           // GROUP_UPDATE_FLAG_MAX_HP
+
+    partyMemberState.CurrentPower = uint16(player->GetPower(powerType));            // GROUP_UPDATE_FLAG_CUR_POWER
+    partyMemberState.MaxPower = uint16(player->GetMaxPower(powerType));         // GROUP_UPDATE_FLAG_MAX_POWER
+    partyMemberState.Level = uint16(player->getLevel());                     // GROUP_UPDATE_FLAG_LEVEL
+    partyMemberState.ZoneId = uint16(player->GetZoneId());                    // GROUP_UPDATE_FLAG_ZONE
+    partyMemberState.PositionX = uint16(player->GetPositionX());                 // GROUP_UPDATE_FLAG_POSITION
+    partyMemberState.PositionY = uint16(player->GetPositionY());                 // GROUP_UPDATE_FLAG_POSITION
+    partyMemberState.PositionZ = uint16(player->GetPositionZ());               // GROUP_UPDATE_FLAG_POSITION
+
+    partyMemberState.VehicleSeat = player->GetVehicle() ? uint32(player->GetVehicle()->GetVehicleInfo()->SeatID[player->m_movementInfo.transport.seat]) : 0;
+
+    partyMemberState.PhaseShiftFlags = uint32(phases.empty() ? 8 : 0);
+    partyMemberState.PhaseCount = uint32(phases.size());
+    for (std::set<uint32>::const_iterator itr = phases.begin(); itr != phases.end(); ++itr)
+        partyMemberState.Phases.push_back(uint16(*itr));
 
+    uint32 auraCounter = 0;
     for (uint8 i = 0; i < MAX_AURAS; ++i)
     {
         if (AuraApplication const* aurApp = player->GetVisibleAura(i))
         {
-            auramask |= (uint64(1) << i);
+            WorldPackets::Party::PartyMemberState::Aura aura;
 
-            data << uint32(aurApp->GetBase()->GetId());
-            data << uint16(aurApp->GetFlags());
+            aura.SpellId = uint32(aurApp->GetBase()->GetId());
+            //todo: Scalings
+            aura.EffectMask = uint32(aurApp->GetEffectMask());
 
             if (aurApp->GetFlags() & AFLAG_SCALABLE)
             {
                 for (uint32 i = 0; i < MAX_SPELL_EFFECTS; ++i)
                 {
                     if (AuraEffect const* eff = aurApp->GetBase()->GetEffect(i))
-                        data << int32(eff->GetAmount());
+                        aura.Scales.push_back(float(eff->GetAmount()));
                     else
-                        data << int32(0);
+                        aura.Scales.push_back(float(0));
                 }
             }
+            auraCounter++;
         }
     }
+    partyMemberState.AuraCount = auraCounter;                    // GROUP_UPDATE_FLAG_AURAS
```

Comment:
this is not needed at all. when writing length to the packet write packet << uint32(partyMemberState.AuraList???.size());`